frontend/console/src/features/timeline/timeline.utils.ts

```diff
@@ -22,4 +27,19 @@ const eventTextColorMap: Record<string, string> = {
   '': 'text-gray-500',
 }
 
-export const eventTextColor = (event: Event) => eventTextColorMap[event.entry.case || '']
+export const eventTextColor = (event: Event) => {
+  if (isError(event)) {
+    return 'text-red-500'
+  }
+  return eventTextColorMap[event.entry.case || '']
+}
+
+const isError = (event: Event) => {
+  if (event.entry.case === 'call' && event.entry.value.error) {
+    return true
+  }
+  if (event.entry.case === 'ingress' && event.entry.value.error) {
+    return true
+  }
```

Comment:
`if (event.entry.value.error && ['call', 'ingress'].includes(event.entry.case))`? Or is there some nullability handling this doesn't work for hehe

Reply:
Yeah it's the typical "make sure this is a call, then you can read the value.error" stuff :)